package.json

```diff
@@ -27,11 +27,17 @@
     "autosize": "^4.0.0",
     "babel-core": "^6.26.0",
     "babel-preset-es2015-rollup": "^3.0.0",
+<<<<<<< HEAD
```

Comment:
It seems like there are some merge conflicts here. I feel like they should be removed.